### PR TITLE
Add UpdateText for BusyInfo

### PR DESCRIPTION
**Pull request by Kvaz1r**
Fix for  [14743 - Add wxBusyInfo::UpdateText()](http://trac.wxwidgets.org/ticket/14743)

If those changes are appropriate I'll update documentation also.

### CODE REVIEW

File: include/wx/generic/busyinfo.h [hunk 1]
```diff
@@ -37,12 +37,15 @@ class WXDLLIMPEXP_CORE wxBusyInfo : public wxObject
         Init(wxBusyInfoFlags().Parent(parent).Label(message));
     }
 
+    void UpdateText(const wxString& str, bool is_markup = true);
```
Comment (vadz) [inline include/wx/generic/busyinfo.h]: Sorry, I don't think bool `is_markup` makes for a good API.

We might just take `wxBusyInfoFlags` and use either its `Text()` or `Label()`, depending on which one is not empty.

Or, maybe simpler, we could have `UpdateText()` and `UpdateLabel()`, with the understanding that the former takes markup and the latter uses plain text only.

File: include/wx/generic/busyinfo.h [hunk 2]
```diff
@@ -37,12 +37,15 @@ class WXDLLIMPEXP_CORE wxBusyInfo : public wxObject
...
+
     virtual ~wxBusyInfo();
 
 private:
     void Init(const wxBusyInfoFlags& flags);
 
     wxFrame *m_InfoFrame;
+    wxControl *m_Text;
```
Comment (vadz) [inline include/wx/generic/busyinfo.h]: You need to add forward declaration of `wxControl` to fix the CI builds.

Also, even though the variable above uses a wrong naming convention, for some reason, let's still follow the convention for this one and call it `m_text`.

**Review by vadz — COMMENTED** [review]
Thanks, this looks good, but let's please avoid using `bool` parameters in the API (see this [guideline](https://www.wxwidgets.org/develop/coding-guidelines/#no_bool_params)) and, of course, fix the build.

File: include/wx/generic/busyinfo.h (hunk 1 above)
Comment (Kvaz1r) [reply include/wx/generic/busyinfo.h]: Yeah, two functions seems to me even better. Fixed.

**Comment by vadz** [comment]
Thanks! Looks good to me and I'll merge it once the CI builds finish.

**Comment by vadz** [comment]
Merged in 16b08c6ef9df2d93c05c3c94e862ca3dc08009c2 (with a copy-pasto in the "Closes" line, which unfortunately resulted in using a completely mangled URL).

**Pull request closed without merging**